acceptance-tests/tests/workflows/secretsFlow.spec.ts

```diff
@@ -23,52 +37,39 @@ describe('Secrets Flow', () => {
     testState.cleanup();
   });
 
-  describe('hs secrets add', () => {
-    it('should create a new secret', async () => {
-      await testState.cli.executeWithTestConfig(
-        ['secrets', 'add', SECRET.name],
-        [SECRET.value, ENTER]
-      );
-    });
-  });
+  it('should create a new secret', async () => {
+    await testState.cli.executeWithTestConfig(
+      ['secrets', 'add', SECRET.name],
+      [SECRET.value, ENTER]
+    );
 
-  describe('hs secrets list', () => {
-    it('should list the secret', async () => {
-      await expect
-        .poll(() => testState.cli.executeWithTestConfig(['secrets', 'list']), {
-          interval: 1000,
-          timeout: 10000,
-        })
-        .toContain(SECRET.name);
-    });
+    await waitForSecretsListToContainSecret(testState);
   });
 
-  describe('hs secrets update', () => {
-    it('should update the existing secret', async () => {
-      await testState.cli.executeWithTestConfig(
-        ['secrets', 'update', SECRET.name],
-        ['a different secret value', ENTER]
-      );
-    });
-  });
+  it('should update the existing secret', async () => {
+    // Wait for the secret to exist before updating it
+    await waitForSecretsListToContainSecret(testState);
```

Comment:
This test kept failing because the secret didn't exist.  Adding a polling check in to make sure it exists before attempting to run the `hs secrets update`